tests/ovp/zend/tests/Test/ZendClientTester.php

```diff
@@ -263,6 +263,52 @@ public function testResponseProfileUnNamed() {
 		$this->assertTrue(isset($result->relatedObjects[1]));
 		$this->assertTrue($result->relatedObjects[0]->objects[0]->id === $entry->userId);
 	}
+
+	public function testMultiLingualObject() {
+		$this->_client->setLanguage('multi');
+		$entry = new Kaltura_Client_Type_BaseEntry();
+		$entry->description = "test multiling";
+		$entry->tags = "testmulti";
+		$nameEn = new Kaltura_Client_Type_MultiLingualString();
+		$nameEn->language = 'EN';
+		$nameEn->value = "Test Entry";
+		$nameEs = new Kaltura_Client_Type_MultiLingualString();
+		$nameEs->language = 'ES';
+		$nameEs->value = "Entrada de prueba";
+		$entry->multiLingual_name = [
+			$nameEn,
+			$nameEs,
+		];
+		
+		$newEntry = $this->_client->baseEntry->add($entry, Kaltura_Client_Enum_EntryType::MEDIA_CLIP);
+
+		$this->assertTrue(empty($newEntry->name));
+		$this->assertTrue(is_array($newEntry->multiLingual_name));
+		$this->assertTrue(empty($newEntry->description));
+		$this->assertTrue(is_array($newEntry->multiLingual_description));
```

Comment:
you didn't add a description, why would it be an array?
you expect it to be an empty array? 
I guess that's ok, this is how other arrays on client objects behave, right?

Reply:
I added description, and since I'm calling with language 'multi' - I expect that description to be set in the default language (EN??)
anyway, the assert passes.

regarding other clients - tests are specific to a client type, so it doesn't really matter but it should behave the same.